src/client/driver/driver.js

```diff
@@ -638,7 +638,14 @@ export default class Driver {
 
         const customCommandHandler = this.customCommandHandlers[command.type];
 
-        return customCommandHandler && customCommandHandler.isExecutableInTopWindowOnly;
+        if (!customCommandHandler)
+            return false;
+
+        const isHandlerExecutableInTopWindowOnly = customCommandHandler.isExecutableInTopWindowOnly;
+
+        return typeof isHandlerExecutableInTopWindowOnly === 'function' ?
+            isHandlerExecutableInTopWindowOnly(command) :
```

Comment:
It looks really hacky. Maybe we can just embed the `isExecutableInTopWindowOnly` in the `command`, not in the handler?

Reply:
Something like the following?
```
return typeof command.isExecutableInTopWindowOnly === 'boolean' ?
            command.isExecutableInTopWindowOnly :
            customCommandHandler && customCommandHandler.isExecutableInTopWindowOnly;
```

Reply:
If `executeInTopWindow` could be a function now, isn't better to make this a function always?
I mean smth like this:
```JS
setCustomCommandHandlers (command, handler, executeInTopWindowOnly = lodash.stubFalse) {
        this.customCommandHandlers[command] = {
            isExecutableInTopWindowOnly: executeInTopWindowOnly,
            handler
        };
    }
```

Thus type checks will no be required. It will be a function all time

Reply:
I don't understand why we should use command handlers to retrieve this information. Let's add the `executableInTopWindowOnly` flag that defaults to `false` to the `CommandBase`, and use this check in the driver:
```
_isExecutableInTopWindowOnly (command) {
        return command.executableInTopWindowOnly || isExecutableInTopWindowOnly(command);
    }
```